src/components/ScenarioParameters/components/ScenarioActions/ScenarioActions.spec.js

```diff
@@ -141,6 +142,8 @@ describe('Test scenario buttons when scenario is running', () => {
     expect(discardChangesButton.Button).not.toBeInTheDocument();
 
     expect(stopRunButton.Button).toBeVisible();
+    expect(getByDataCy('running-state-spinner')).toBeVisible();
+    expect(runningStateLabel.Typography).toBeVisible();
```

Comment:
We should probably also check that the spinner and 'running' label are not visible (or don't exist) when the scenario is not running

Reply:
Good idea!